src/crimson/osd/osdmap_gate.h

```diff
@@ -42,12 +42,11 @@ class OSDMapGate {
     }
   };
 
-  // order the promises in descending order of the waited osdmap epoch,
+  // order the promises in ascending order of the waited osdmap epoch,
```

Comment:
the commit message is not quite accurate,

> [waiting_peering.begin(), waiting_peering.upper_bound(epoch)]

it should be

> [waiting_peering.begin(), waiting_peering.upper_bound(epoch))

Reply:
Um...it seems that "upper_bound(epoch)" is included.